developer_manual/general/backporting.rst

```diff
@@ -21,3 +21,39 @@ We mostly consider bug fixes for back porting. Occasionally, important changes t
 .. note:: Before each patch release there is a freeze to be able to test everything as a whole without pulling in new changes. This freeze is announced on the `owncloud-devel mailinglist <https://mailman.owncloud.org/pipermail/devel/>`_. While this freeze is active a backport isn't allowed and has to wait for the next patch release.
 
 The QA team will try to reproduce all the issues with the X.Y.Z-next-maintenance milestone on the relevant release and verify it is fixed by the patch release (and doesn't cause new problems). Once the patch release is out, the post-fix -next-maintenance is removed and a new -next-maintenance milestone is created for that series.
+
+Steps
+-------
+
+Because pushing directly to particular branches of ownCloud like stable versions is forbidden,
+(eg. no direct push to origin/stable-xx) you need to create your own remote branch and set the 
+base for this PR correctly.
+Following example is a backport of commit ``123456`` in PR ``789`` to branch ``stable10``.
+Please adopt the referenced commit to be backported and the target branch accordingly.
+Use ``sudo`` if necessary.
+
+1. Get target branch ``stable10`` up to date from GitHub
+
+.. code-block::
+
+  git fetch origin
+  git checkout stable10
+  git pull
+
+
+2. Create a local branch from the target and cherry-pick your commit
+
+.. code-block::
+
+  git checkout -b stable10-fix_for_789
+  git cherry-pick 123456
+  (add more cherry-picks here if necessary)
+  git push
```

Comment:
git push --set-upstream origin stable10-fix_for_789

We can give the actual command here (I just tried it to get the right syntax).

Reply:
updated